js/helper.js

```diff
@@ -105,13 +105,25 @@ function requestUserData(notify, callback) {
       if (notify === true && currentData.refreshInterval != 0){
           var notified = false;
           if (nbReviews > 0 && nbReviews != currentData.nbReviews) {
-            createNotification("You have " + nbReviews +" reviews available.", "https://www.wanikani.com/review", "reviews");
-            notified = true;
+            if (currentData.nbReviews === 1) {
+            // special case of a single review
+              createNotification("You have " + nbReviews +" review available.", "https://www.wanikani.com/review", "reviews");
```

Comment:
No need to duplicate everything. :) You can only change the word with something like `var reviewTxt = (nbReviews > 1) ? "reviews" : "review";` then use it in the message.